examples/tenant_tutorial/tenant_tutorial/urls_public.py

```diff
@@ -1,6 +1,6 @@
-from django.conf.urls import patterns, url
```

Comment:
The library is intended to work in full extent (including tutorial) in lower versions as well.